app/src/main/java/com/cobresun/factfulnewsandroid/TabFragment.kt

```diff
@@ -24,27 +26,44 @@ import timber.log.Timber
 
 class TabFragment(index: Int) : Fragment() {
     var tabIndex = index
+    var articles : List<Article>? = null
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+        if(activity != null){               // Afterwards keep displaying already initialized articles
+            articles?.let { showArticles(it) }
+        }
+    }
 
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-
         val retrofit = Retrofit.Builder()
             .baseUrl("https://factfulnews.herokuapp.com")
             .addConverterFactory(GsonConverterFactory.create())
             .build()
 
-        val api = retrofit.create(ApiService::class.java)
-        api.fetchArticles(CategoryUtils.categories[tabIndex]).enqueue(object: Callback<FetchResponse> {
-            override fun onResponse(call: Call<FetchResponse>, response: Response<FetchResponse>) {
-                showArticles(response.body()!!.articles)
```

Comment:
I notice this PR needs to rebase on top of your own previous commit LOL https://github.com/Cobresun/factfulnewsAndroid/pull/13 also one that I put in, so make sure to rebase and then double check everything works ...

Reply:
oh yikes